modules/ipi-install-configuring-bios-for-worker-node.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_bare_metal_ipi/ipi-install-configuration-files.adoc
+[id="configuring-bios-for-worker-node_{context}"]
+
+= Configuring BIOS for worker node
+
+During the installation process, the BIOS of the worker node can be configured
+by the following steps.
+
+.Procedure
+. Create manifests
+. Modify the BMH file corresponding to the worker
++
+----
+$ vim clusterconfigs/openshift/99_openshift-cluster-api_hosts-3.yaml
+----
+. Add BIOS configuration in `spec`:
++
+----
+spec:
+  firmware:
+    simultaneousMultithreadingEnabled: true
+    sriovEnabled: true
+    virtualizationEnabled: true
+----
++
+[NOTE]
+====
+. Only support these three BIOS configurations. See the
+link:https://github.com/metal3-io/baremetal-operator/blob/master/docs/api.md#firmware[BMH documentation] for details.
```

Comment:
We are not allowed to link to upstream repos as sources. So the link needs to be removed and details brought downstream.